src/components/layout/Footer.module.scss

```diff
@@ -0,0 +1,35 @@
+@import './../../assets/styles/vars.scss';
```

Comment:
IMO, we should put all components under `components` folder, we have too many sub-folders like `layout`, `layouts` and `pages`, which could make it hard for readers to understand the structure of the source codes, and the functionalities of the components.